packages/react-api/src/hooks/gpu-filter-utils.js

```diff
@@ -0,0 +1,88 @@
+import { _buildFeatureFilter, _FilterTypes } from '@carto/react-core/';
+
+// Don't change this value to maintain compatibility with builder
+export const MAX_GPU_FILTERS = 4;
+
+function getFiltersWithoutTimeType(filters) {
+  const filtersWithoutTimeType = {};
+  let timeFilter = {};
```

Comment:
```suggestion
  let timeFilter = null;
```

Si pones timeFilter como null aquí, te ahorras ese Object.keys length cada vez que quieres comprobar si timeFilter existe en `getFilterRange`, `getFilterValueUpdateTriggers` y `getFilterValue`.

Reply:
Done